src/test/java/org/pesmypetcare/webservice/controller/communitymanager/ForumRestControllerTest.java

```diff
@@ -131,10 +133,29 @@ public void postMessage() throws Exception {
 
     @Test
     public void deleteMessage() throws Exception {
-        willDoNothing().given(service).createForum(anyString(), any(ForumEntity.class));
+        willDoNothing().given(service).deleteMessage(anyString(), anyString(), anyString(), anyString(), anyString());
+        mockMvc.perform(
+            delete(BASE_URL + parentGroup + "/" + forumName).header("token", myToken)
+                .param("creator", creator).param("date", creationDate)).andExpect(status().isNoContent());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "creator" appears 5 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5633182)